letsencrypt/client/display.py

```diff
@@ -30,6 +31,8 @@ def redirect_by_default(self):
 
```

Comment:
Thanks! The whole display situation (python-dialog compatibility / which methods should "display" provide) needs to be decided/taken care of... thanks for the quick fix.